internal/metrics/mcp_metrics.go

```diff
@@ -78,6 +78,7 @@ type mcpStatusType string
 
 const (
 	mcpStatusSuccess mcpStatusType = "success"
+	mcpStatusFailed  mcpStatusType = "failed"
```

Comment:
This seems to not be used anymore, but the idea was that when there is an application error, we use this code instead, right?